src/commands/Slots.js

```diff
@@ -66,89 +66,91 @@ class Slots extends Command {
 		result += `\\\u2197     ${emojis[frst + 1]}  ${emojis[scnd + 1]}  ${emojis[thrd + 1]}     \\\u2196`;
 		result += `\n--------------------------\n\n\n`;
 
-		message.channel.createMessage({
-			embed: {
-				url: 'https://dynobot.net/',
-				description: `Spinning the wheels of fortune for [${message.member.username}](https://dynobot.net)...`,
-				color: 1259143,
-				thumbnail: {
-					url: 'https://s18.postimg.org/hmpvac5qh/dynoslots_spin.gif',
+		return new Promise(res => {
+			message.channel.createMessage({
+				embed: {
+					url: 'https://dynobot.net/',
+					description: `Spinning the wheels of fortune for [${message.member.username}](https://dynobot.net)...`,
+					color: 1259143,
+					thumbnail: {
+						url: 'https://s18.postimg.org/hmpvac5qh/dynoslots_spin.gif',
+					},
 				},
-			},
-		}).then(newmsg => {
-			if (this.checkJackpot(gameid, bonus)) {
-				setTimeout(() => {
-					message.channel.editMessage(newmsg.id, {
-						embed: {
-							title: 'JACKPOT!',
-							url: 'https://dynobot.net/',
-							color: 13496340,
-							description: `${result}**Woah, thats a Jackpot!** You were lucky and reached the best result in this game possible!\nNow thats a thing to screenshot and share with your friends, [${message.member.username}](https://dynobot.net)!`,
-							footer: {
-								text: `Game played by ${message.member.username}#${message.member.discriminator}`,
-								icon_url: message.member.avatarURL,
+			}).then(msg => {
+				if (this.checkJackpot(gameid, bonus)) {
+					setTimeout(() => {
+						res(message.channel.editMessage(msg.id, {
+							embed: {
+								title: 'JACKPOT!',
+								url: 'https://dynobot.net/',
+								color: 13496340,
+								description: `${result}**Woah, thats a Jackpot!** You were lucky and reached the best result in this game possible!\nNow thats a thing to screenshot and share with your friends, [${message.member.username}](https://dynobot.net)!`,
+								footer: {
+									text: `Game played by ${message.member.username}#${message.member.discriminator}`,
+									icon_url: message.member.avatarURL,
+								},
+								thumbnail: {
+									url: 'https://s17.postimg.org/w4v2gei1r/tumblr_mh79cf_ON001r88gpzo1_400.gif',
+								},
 							},
-							thumbnail: {
-								url: 'https://s17.postimg.org/w4v2gei1r/tumblr_mh79cf_ON001r88gpzo1_400.gif',
+						}));
+					}, 2000);
+				} else if (this.checkTriple(frst, scnd, thrd, gameid)) {

```

Comment:
Can we use readable arguments, we don't really need to shorten already short words. first, second, third. Same with other instances below.